services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/ApplicationConfig.java

```diff
@@ -60,9 +61,8 @@
     /**
      * Exposes a Vert.x instance as a Spring bean.
      * <p>
-     * This method creates new Vert.x default options and invokes
-     * {@link VertxProperties#configureVertx(VertxOptions)} on the object returned
-     * by {@link #vertxProperties()}.
+     * This method creates new Vert.x default options and invokes {@link VertxProperties#configureVertx(VertxOptions)}
```

Comment:
Is there a reason why several of the existing _javadocs_ (several other places in this class) have been reformatted? If not could you please revert back to their original state and they were already formatted.

Reply:
done